### PR TITLE
Update istio-cni to use logAsJson Flag

### PR DESCRIPTION
**Pull request by erismaster**
The istio-cni chart was missing the logAsJson flag/bool in the chart template. This PR adds that flag, similar to other charts such as istiod.

Signed-off-by: Derrik Campau <dcampau@vmware.com>

**Please provide a description of this PR:**
The helm chart template for istio-cni was not allowing for users to customize if istio-cni should log as JSON output. This PR fixes that :)

### CODE REVIEW

**Comment by erismaster** [comment]
I am waiting on a company process to finish up for the CLA to pass.

**Comment by howardjohn** [comment]
lgtm once we get the CLA covered

**Review by LaikaN57 — APPROVED** [review]

**Review by zirain — APPROVED** [review]
lgtm, thanks

**Comment by erismaster** [comment]
@jacob-delgado or @costinm Apologies for a direct ping, but was wondering if this could get the last lgtm to merge :) 
Thanks!

**Review by jacob-delgado — APPROVED** [review]

**Pull request merged**
